### PR TITLE
Accessibility fix part 2

### PR DESCRIPTION
**Pull request by peiwentian**
Accessibility fix.
Display error message on 'Network Configuration' window

### CODE REVIEW

**Comment by peiwentian** [comment]
The error message will be displayed at the bottom of the TextBox
image

**Comment by peiwentian** [comment]
Tested the following flow

1. Prepare Environment (boot into CloudBuilder)
2. Install ASDK flow, able to reach to final screen to generate the deployment script.
3. Reboot flow (reboot back into windows server)

File: Deployment/asdk-installer.ps1 [hunk 1]
```diff
@@ -2680,6 +2727,18 @@ $syncHash.Control_Creds_Cbx_Idp.Add_SelectionChanged({
 })
 
 $syncHash.Control_Creds_Tbx_AADTenant.Add_TextChanged({
+	$fiedlValue = $syncHash.Control_Creds_Tbx_AADTenant.Text
```
Comment (donovanmms) [inline Deployment/asdk-installer.ps1:2730]: typo (same typo is in a bunch of different places):

$fiedlValue -> $fieldValue

Comment (peiwentian) [reply Deployment/asdk-installer.ps1]: Thanks for catching that typo! Made the changes

**Review by donovanmms — APPROVED** [review]
Looks good now

**Pull request merged**